### ISSUE TITLE
Alfred/QuickLook doesn't show website preview

### ISSUE THREAD

**Issue by adamkiss** [spamwax/alfred-pinboard-rs#6]
Hello,

Not sure if I need a specific (newer?) version of macos and/or some quicklook plugin, but I'm not getting the pin/website preview, I only see a safari icon and pin title.

Alfred: up-to-date
alfred-pinboard-rs: up-to-date
macOs: 10.11.6

### Expected:

See preview of github page

### Actual:

![](https://dl.dropboxusercontent.com/s/rerl9hjy2uz3h2p/Screenshot%202018-04-30%2015.02.19.png)

Let me know if you need anything else, and thank you!

**Comment by spamwax** [comment]
I am having the same issue.
According to [this post](https://www.alfredforum.com/topic/11413-lost-ability-to-quicklook-urls-in-macos-1012-bug-in-macos-sierra/?tab=comments#comment-59696), this is related to a bug in macOS 10.12, though I am not sure why you are getting the same result on 10.11

**Comment by adamkiss** [comment]
@spamwax Yes, I am running El Capitan 10.11.6, specifically build 15G1421, which is I think two security patches behind actually latest.
